### DISCUSSION TITLE
当我升级到 v4.16.13 时 rc-menu 组件在 mode="horizontal" 提示错误 😭

### DISCUSSION THREAD

**Discussion by wangzongming** [ant-design/ant-design#32270] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/27466569/134604946-f551c95b-c73e-4117-a554-26a61b69107f.png)

![image](https://user-images.githubusercontent.com/27466569/134605087-9a33cb38-ca84-4cff-b72a-097f06402e1d.png)

错误里面提示是 mini-store 这个组件报错了，我已经排查了整个 node_modules 中没有重复的 mini-store 文件夹了。
而且也只有在设置了 mode="horizontal"  才会报这个错误，并且我并没有使用 redux/react-redux. 下面是我的代码。

![image](https://user-images.githubusercontent.com/27466569/134605174-5e27a3ad-4d58-40cf-a3d2-80b8c9050ede.png)

你们遇到过这个错误吗？或者是哪些情况下可能会导致这个错误呢？

**Comment by xrkffgg** [comment, marked as answer]
开个 issue ，提供下复现。

**Reply by wangzongming** [reply]
我在 codesandbox 复现不出来，所以很蒙，但是我的 node_modules 又不想完全重新安装一遍。因为有比较多的依赖。八成是和哪个依赖冲突了。
我现在正在看 rc-menu 组件在哪里用了 mini-store

codesandbox 上面虽然复现不出来我那个 bug, 但是那个菜单几乎也用不了，会直接卡死。(代码直接复制文档网站上面的)
https://codesandbox.io/s/ding-bu-dao-hang-antd-4-17-0-alpha-3-forked-yqg3b?file=/index.js

**Reply by xrkffgg** [reply]
直接卡死 像是 csb 的问题

**Reply by wangzongming** [reply]
我本地确实没有卡死的现象（不加 mode="horizontal" 就可以正常出来）

**Reply by wangzongming** [reply]
我排查出来了，原因是旧版的 rc-menu 中有个 subMenu.js 文件，最新版9.0.12 中 subMenu 改成了一个文件夹，但是名字没改，而我是直接把最新文件覆盖旧文件的操作升级，所以导致 rc-menu 代码中的引用出了问题（引用了js文件而不是文件夹中的index.js）...  🤣这该死的巧合.
